scripts/io_stability/profiling.yaml

```diff
@@ -0,0 +1,218 @@
+apiVersion: v1
```

Comment:
from where did we get this? Will this chnage with service framework versions?

Reply:
@priyankaborawake

Reply:
We got this from Vinoth from Dev team, This wont change with service framework version.